context/app/static/js/components/Routes/Routes.jsx

```diff
@@ -101,7 +101,7 @@ function Routes({ flaskData }) {
   if (urlPath.startsWith('/browse/publication/')) {
     return (
       <Route>
-        <Publication publication={entity} vignette_json={vignette_json} />
+        <Publication publication={entity} vignette_json={vignette_json} visLiftedUUID={vis_lifted_uuid} />
```

Comment:
Should we consider adding a context to avoid needing to pass this prop down? 
- The `Publication` component itself doesn't use this value except passing it down to `Files`
- The `VisLIftedUUID` value does not change on the page, and would cause a rerender of the whole `Publication` even if it did;
- Creating a context and a corresponding `use` hook for it would help keep the variable visible only where it's used

Reply:
Issue regarding passing flask data by context https://github.com/hubmapconsortium/portal-ui/issues/3088